kits/democracy/readme.md

```diff
@@ -25,6 +25,20 @@ Then just:
 npm run deploy:rinkeby
 ```
 
+## Permissions
+
+| App           | Permission       | Grantee | Manager |
+|---------------|------------------|---------|---------|
+| Voting        | CREATE_VOTES     | Any     | Voting  |
+| Voting        | MODIFY_QUORUM    | Voting  | Voting  |
+| Voting        | MODIFY_SUPPORT   | None    | Burned  |
+| Vault         | TRANSFER         | Finance | Voting  |
+| Finance       | CREATE_PAYMENTS  | Voting  | Voting  |
+| Finance       | EXECUTE_PAYMENTS | Voting  | Voting  |
+| Finance       | DISABLE_PAYMENTS | Voting  | Voting  |
+| Token Manager | ASSIGN           | Voting  | Voting  |
+| Token Manager | REVOKE_VESTINGS  | Voting  | Voting  |
```

Comment:
Ah, actually, it'd be nice to include the roles of the `ACL`, `Kernel`, and `EVMScriptRegistry` here too :).